### ISSUE TITLE
How to limit max subscriptions per user

### ISSUE THREAD

**Issue by rajanmodi30** [emqx/emqx#2908]
#### Environment

- OS: Ubuntu 16.04
- Erlang/OTP: 
- EMQ: 3.2.3

#### Description

I am trying to limit max subscriptions per user, l could find 'zone.external.max_subscriptions' in the emqx.conf file, however I cant figure out whether this is for a particular user or for all the users. How can I do it for user or mountpoint basis? Can I configure this parameter for a particular mountpoint or user? And how?

```
## Maximum number of subscriptions allowed, 0 means no limit.
##
## Value: Number
zone.external.max_subscriptions = 100
```

**Comment by HJianBo** [comment]
It only limits max count topics per user on the specified zone (i.g: `external`).

After reviewing codes, it not works on the current version (3.2.3). I'll try to fix it later

**Comment by turtleDeng** [comment]
Thanks for your feedback, we have fixed it in v3.2.4.
